src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -920,4 +920,10 @@ To use a custom JDK path for a command line build, set the 'JavaSdkDirectory' MS
 {1} - A Filename.
     </comment>
   </data>
+  <data name="XA_Fixup_Designer_Error" xml:space="preserve">
+    <value>Could not find an 'AndroidResource' for '{0}'. To Fix this issue add the required 'AndroidResource' item to your project.</value>
```

Comment:
I'm not sure this message is "actionable" enough?

Given that we're (potentially) emitting an XA7000 error code, at minimum we should have a `Documentation/guides/messages/xa7000.md` file as part of this PR describing the scenario and how to fix it.

Can more information be provided?  For example, what assembly/NuGet/etc. is missing the `@(AndroidResource)` value?  Maybe suggest that the "offending" assembly/package needs to provide a `.aar` file containing the specified resource?

Reply:
I added some docs in the latest commits. 
Note that the number is now 8000. This is so we can cover the same error with the same docs. 
Under MSBuild we will get XA8000 but the linker (in Release mode) will produce  IL8000. This is why I chose 8000. Our current linker errors are XA2xxx . The ILLink code only allows "custom" codes as long as they are over 7000.